listenbrainz_spark/utils.py

```diff
@@ -96,3 +97,35 @@ def save_parquet(df, path):
         df.write.format('parquet').save(path, mode='overwrite')
     except Py4JJavaError as err:
         raise Py4JJavaError('Cannot save parquet to {}: {}\n'.format(path, type(err).__name__), err.java_exception)
+
+def create_dir(path):
+    """ Creates a directory in HDFS.
+
+        Args:
+            path (string): Path of the directory to be created.
+
+        Note: >> Caller is responsibe for initializing HDFS connection.
+              >> The function do not throws an error if the directory already exists.
+    """
+    hdfs_connection.client.makedirs(path)
+
+def delete_dir(path):
+    """ Deletes a directory recursively from HDFS.
+
+        Args:
+            path (string): Path of the directory to be deleted.
+
+        Note: Caller is responsible for initializing HDFS connection.
+    """
+    hdfs_connection.client.delete(path, recursive=True)
+
+def get_status(path):
+    """ Checks the status of a directory in HDFS. May be used to check if a directory
```

Comment:
What is the return value of this?

Reply:
https://hdfscli.readthedocs.io/en/latest/api.html#hdfs.client.Client.makedirs

Reply:
If you try to get the return value in a variable and print, the value would always be 'None'

Reply:
http://hadoop.apache.org/docs/r1.0.4/webhdfs.html#FileStatus

Reply:
If the directory does not exist, it will throw HDFS error. If we catch an error, it means no directory with such name exists, and hence we can call create_dir because  this -https://hdfscli.readthedocs.io/en/latest/api.html#hdfs.client.Client.makedirs  
does not raise an error if we try to create a directory with a path that already exists.

Reply:
You should put that into the docstrings, not to me. :)